modules/dynunet_pipeline/create_network.py

```diff
@@ -7,22 +7,38 @@
 
 
 def get_kernels_strides(task_id):
+    """
+    This function is only used for decathlon datasets with the provided patch sizes.
+    When refering this method for other tasks, please ensure that the patch size for each spatial dimension should
+    be divisible by the product of all strides in the corresponding dimension.
+    In addition, the minimal spatial size should have at least one dimension that has twice the size of
+    the product of all strides. For patch sizes that cannot find suitable strides, an error will be raised.
+
+    """
     sizes, spacings = patch_size[task_id], spacing[task_id]
+    input_size = sizes
     strides, kernels = [], []
-
+    min_patch_size = [1, 1, 1] * len(sizes)
```

Comment:
This variable doesn't appear used anywhere except to update itself?

Reply:
Thanks Eric, I forgot to remove it.